src/scala/com/github/johnynek/bazel_deps/Resolver.scala

```diff
@@ -173,6 +181,7 @@ class Resolver(servers: List[MavenServer], resolverCachePath: Path) {
       val artifact = a.getArtifact
       MavenCoordinate(MavenGroup(artifact.getGroupId),
         MavenArtifactId(artifact.getArtifactId),
+        Classifier(artifact.getClassifier),
```

Comment:
is this always non-null? I guess the others are non-null, but I never used it before.

Reply:
I had to do some digging, but it appears so, yep: 

> The classifier or an empty string if none, never null.
-- http://sonatype.github.io/sonatype-aether/apidocs/org/sonatype/aether/artifact/Artifact.html#getClassifier()